src/stores/__tests__/UserStore.spec.ts

```diff
@@ -8,29 +8,27 @@ import { encodeJWT } from '@/__tests__/helpers';
 import { useUserStore } from '../UserStore';
 
 describe('userStore', () => {
-  let userStore: ReturnType<typeof useUserStore>;
-
   beforeEach(() => {
     setActivePinia(createPinia());
     vi.useFakeTimers({
       now: new Date('2024-01-01T00:00:00Z'),
     });
-
-    userStore = useUserStore();
```

Comment:
Looks like `userStore` is being used in every test case, why moving this declaration to each specific `it`?

Reply:
As a general rule of thumb, I like to instantiate everything on each test, but in this particular case I needed to mock the console before initiating the store, so I did the refactor of the tests
https://github.com/RedHatProductSecurity/osim/blob/8b6a90542a6f0f0e27293b0c13ffccef795b0fcd/src/stores/__tests__/UserStore.spec.ts#L87-L89